commodore/component/__init__.py

```diff
@@ -141,6 +142,20 @@ def checkout(self):
         except BadName as e:
             raise RefError(f"Revision '{self.version}' not found in repository") from e
 
+    def render_jsonnetfile_json(self, component_params):
+        """
+        Render jsonnetfile.json from jsonnetfile.jsonnet
+        """
+        jsonnetfile_jsonnet = self._dir / "jsonnetfile.jsonnet"
+        if jsonnetfile_jsonnet.is_file():
+            # pylint: disable=c-extension-no-member
+            output = _jsonnet.evaluate_file(
+                str(jsonnetfile_jsonnet),
+                ext_vars=component_params.get("jsonnetfile_parameters", {}),
+            )
+            with open(self._dir / "jsonnetfile.json", "w") as fp:
```

Comment:
What if both `jsonnetfile.jsonnet` and `jsonnetfile.json` exist (and are checked in into git)? Can we print a warning for that case?